### PR TITLE
58 spar annotation

### PR DESCRIPTION
**Pull request by ellaparke**

### CODE REVIEW

**Review by apjmason — CHANGES_REQUESTED** [review]
@ellaparke after reading this, I think we should change the head word to "tacking-boom" and make this an annotation for beitiass in particular, not just ass in general. We can define general "spar" as ass in the definition, but it's the tacking-boom that you are most concerned with here and also what we have modeled on our ship in VR. 

Can you make those changes, referencing the entries for tacking boom on the [Ship Terminology spreadsheet](https://docs.google.com/spreadsheets/d/1SdY51aqCSdwGvNHDfiLJjZuJCulogexdHsut1iaPg-M/edit?gid=1259887355#gid=1259887355)?

File: annotations/spar.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<div align="center">
+  
+<img width="470" alt="Screenshot 2024-12-04 at 8 53 06 pm" src="https://github.com/user-attachments/assets/949c9959-9fc1-4166-88b5-47775688de48">
+  
+  Gotland Picture Stone (The Picture Stone from Hunninge at Klinte, Viking Archaeology)
+
+</div>
+
+  Use of the spar reduces strain on the side of a ship, by spreading the pressure from the sail over to the opposite side, where the forces can be spread over several frames (Bishcoff, Vibeke). 2). The use of beitiass is also known from iconographie sources as on the Gotland picture stone from Klinte Hunninge and from ethnographic boats from northwestern Iceland (Imer 2004, 58; Andersen and Andersen
```
Comment (apjmason) [inline annotations/spar.md:16]: This is well written, but the Imer 2004 and Andersen and Andersen references are not included below
```suggestion
  Use of the spar reduces strain on the side of a ship, by spreading the pressure from the sail over to the opposite side, where the forces can be spread over several frames (Bishcoff 2017, 2). The use of the beitiáss is also known from iconographic sources as on the Gotland picture stone from Klinte Hunninge and from ethnographic boats from northwestern Iceland (Imer 2004, 58; Andersen and Andersen
```

File: annotations/spar.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1989, 192-19)
+
+---
+
+  Bischoff, Vibeke. “Viking-Age Sails: Form and Proportion.” Journal of Maritime Archaeology 12, no. 1 (2017): 1–24. 
+  Hale, John R. “The Viking Longship.” Scientific American 278, no. 2 (1998): 56–63.
```
Comment (apjmason) [inline annotations/spar.md:22]: Add a space between references to preserve formatting.

Add Imer 2004 and Andersen and Andersen sources

**Review by apjmason — CHANGES_REQUESTED** [review]

**Review by ellaparke — COMMENTED** [review]
Changes have been made except for the one of the sources as I cannot find it.

**Review by apjmason — APPROVED** [review]
Updated and now surpassed

**Pull request merged**